### PR TITLE
[cni-cilium] Preserve default tunnel port 8472 for virtualization workloads

### PR DESCRIPTION
**Pull request by kvaps**
## Description

This PR recofigures cilium to use non-standard port 8472 for tunnel in case of virtualization module enabled.

## Why do we need it, and what problem does it solve?

Due to [`CVE-2019-13119`](https://cilium.io/blog/2019/07/01/cve-2019-13119/), cilium in tunnel mode drops all packets on the same port used for vxlan tunnel, which makes not possible to run nested kubernetes clusters using same port for tunneling:
```
xx drop (Encapsulation traffic is prohibited) flow 0xcab0685a to endpoint 0, identity 3087->2: 10.16.209.20:39258 -> 192.168.122.124:8472 udp
```

ref:
- https://github.com/cilium/cilium/pull/8434
- https://github.com/cilium/cilium/pull/20115

## What is the expected result?

When `virtualization` module enabled, cilium starts using 8469 port instead of 8472, to allow reusing this port by nested Kubernetes clusters. 

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: cni-cilium
type: fix
summary: Preserve default tunnel port `8472` for virtualization workloads.
impact: Short network downtime for virtualization enabled clusters.
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @kvaps. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @kvaps. I've started the workflow for label 'deploy/web/test'...

**Review by zuzzas — APPROVED** [review]

File: modules/490-virtualization/docs/CONFIGURATION.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ title: "The virtualization module: configuration"
 {% include module-bundle.liquid %}
 
 > **Note!** Module depends on the [cni-cilium](../021-cni-cilium/) module. Make sure your cluster is deployed with Cilium as the main CNI plugin.
+>
+> If cilium works in tunnel mode, enabling this module will result in short downtime due to reconfiguration of overlay network to a non-standard port.
```
Comment (apolovov) [inline modules/490-virtualization/docs/CONFIGURATION.md:9]:
```suggestion
> If cilium works in tunnel mode, enabling this module will result in short downtime due to reconfiguration of overlay network to a non-standard VXLAN port.
```

File: modules/490-virtualization/docs/CONFIGURATION_RU.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ title: "Модуль virtualization: настройки"
 {% include module-bundle.liquid %}
 
 > **Важно!** Для работы модуля необходим включенный модуль [cni-cilium](../021-cni-cilium/).
+>
+> Если cilium работает в режиме тунеля, включение этого модуля приведет к короткому даунтайму из-за перенастройки оверлейной сети на нестандартный порт.
```
Comment (apolovov) [inline modules/490-virtualization/docs/CONFIGURATION_RU.md:9]:
```suggestion
> Если cilium работает в режиме тунеля, включение этого модуля приведет к короткому даунтайму из-за перенастройки оверлейной сети на нестандартный порт VXLAN.
```

**Review by apolovov — APPROVED** [review]

**Review by yalosev — APPROVED** [review]

**Review by konstantin-axenov — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Failure: cherry-pick commit 44f10371a0ed29ee40bd519db23eebcb1b0af0ef to the branch [release-1.43](https://github.com/deckhouse/deckhouse/tree/release-1.43) failed. See [Job](https://github.com/deckhouse/deckhouse/actions/runs/4196927459) for details.

**Comment by z9r5** [comment]
Cherry-picked into release-1.43

**Pull request merged**